lib/rspec/core/configuration.rb

```diff
@@ -399,6 +420,7 @@ def treat_symbols_as_metadata_keys_with_true_values=(_value)
       # @overload shared_context_metadata_behavior=(value)
       #   @param value [:trigger_inclusion, :apply_to_host_groups] sets the configured behavior
       define_reader :shared_context_metadata_behavior
+
```

Comment:
Please don't add random whitespace :)

Reply:
I noticed there's always an empty line before `define_reader :setting` and `def setting=(value)`, except for this place here.

Since the comment in there is:
```ruby
      define_reader :shared_context_metadata_behavior
      # @see shared_context_metadata_behavior
      def shared_context_metadata_behavior=(value)
```

I figured it would be better to add this empty line, to be consistent. I'll revert this, it's not related to the rest of the changes.